BUILDING.md

```diff
@@ -83,8 +83,9 @@ to clone the gRPC repository at the [latest stable release tag](https://github.c
 > git submodule update --init
 ```
 
-NOTE: The `bazel` build tool uses a different model for dependencies. You only need to worry about downloading submodules if you're building
-with something else than `bazel` (e.g. `cmake`).
+> **Note**
```

Comment:
It would be better if your change focused either on suggesting a better formatting OR making changes to the content of the doc. Like this we have a mixture of both and the formatting changes are distracting from the important stuff.

Reply:
I'm also not convinced that the new formatting in better than the original, but that's obviously just my personal opinion.

Reply:
I tried to put formatting changes in their own commit (e.g. [#31409](https://github.com/grpc/grpc/pull/31409/commits/69a2be75e7d3df98409eb34f43f55d344d8d6155)) but that's probably been lost in the PR. 

I don't have a strong opinion about the new 'note block' format other than it's supported by GitHub. My main intention was to make it more consistent. (To be fair, the doc was 'mostly' consistent - it just had a couple of notes inside code blocks.)

Reply:
Feel free to propose formatting changes / style improvements in a separate PR (where you ONLY make formatting/style/typo fixes), and I'm sure the review of that will be much easier and also much faster to accept/reject.
With a big PR that changes the content AND makes style fixes at the same time it's going to be very difficult to move forward.

Reply:
To be honest, it's somewhat difficult to change content alone here without modifying the layout and style, etc. I've tried to record respective changes in the commit history for the PR but clearly, it's not as easy to browse and interpret this over a more readily digestible PR. I will revert to the original version and try submit PRs accordingly.